orbit/pkg/platform/platform.go

```diff
@@ -0,0 +1,156 @@
+package platform
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/mitchellh/go-ps"
+	"github.com/rs/zerolog/log"
+	gopsutil_process "github.com/shirou/gopsutil/v3/process"
+)
+
+var ErrProcessNotFound = errors.New("process not found")
+
+// Kills a single process by its name
+func KillProcessByName(name string) error {
+	if name == "" {
+		return fmt.Errorf("Invalid argument was provided")
```

Comment:
Maybe could be more specific?
```suggestion
		return fmt.Errorf("process name may not be empty")
```

Reply:
Good, I fixed this one, and the rest of the sanity checks on the functions I've added to platform.go